elifepubmed/utils.py

```diff
@@ -15,10 +16,12 @@ def allowed_tags():
         '<p>', '</p>'
     )
 
+
 def allowed_tag_names():
```

Comment:
Could be:
```
def allowed_tag_names():
    return list(sorted(set([re.sub('[<>/]', '', tag) for tag in allowed_tags()])))
```

Reply:
Yes, I like it. This is only a first pass at linting, and there is always room for improvement!